Packs/ServiceNow/Playbooks/playbook-ServiceNow_-_Ticket_Management.yml

```diff
@@ -0,0 +1,843 @@
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: de59d3d7-4a62-445e-8a1c-8a2bb052435b
+    type: condition
+    task:
+      id: de59d3d7-4a62-445e-8a1c-8a2bb052435b
+      version: -1
+      name: Is the comment defined on the input?
+      description: Checks if the comment is defined on the input.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "19"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.CommentToAdd
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -860,
+          "y": 1160
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 9f2c3b0e-0b6c-45ba-8b92-89c13644a4e4
+    type: collection
+    task:
+      id: 9f2c3b0e-0b6c-45ba-8b92-89c13644a4e4
+      version: -1
+      name: Add a comment to the ticket
+      description: Data collection task to add a comment to the ticket.
+      type: collection
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1080,
+          "y": 1330
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: Add comment to the ticket
+        required: true
+        gridcolumns: []
+        defaultrows: []
+        type: longText
+        options: []
+        optionsarg: []
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: what is the comment for the ticket?
+      description: ""
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: d3c1b2c3-5732-457b-8c1e-fe3c8d43c328
+    type: condition
+    task:
+      id: d3c1b2c3-5732-457b-8c1e-fe3c8d43c328
+      version: -1
+      name: Whether to append a new comment to the ticket for each alert?
```

Comment:
Not clear what you're doing here, what does it mean "for each alert".
Try to give better title and description. (Not the same)

Reply:
Done